src/Pages/ChannelRequests.razor

```diff
@@ -574,9 +585,7 @@
 
     private void HideRejectOrCancelModal()
     {
-        _rejectCancelModalVisible = false;
-        _cancelOrRejectReason = null;
-
+        ResetChannelCancelModal(); 
```

Comment:
I think you could just change the method called in the `Clicked` tag in line 286 and remove this function.

Reply:
true, changing it. Thanks!